com.synology.SynologyDrive.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Synology Drive Client
+Comment=Synology Drive Client
+Exec=synology-drive start
+Icon=me.gregos.SynologyDrive
```

Comment:
this is not the icon you're installing. it should be `com.synology.SynologyDrive`

Reply:
corrected for the next commit